src/main/java/domain/Lotto.java

```diff
@@ -1,29 +1,37 @@
 package domain;
 
-import domain.numbergenerator.NumberGenerator;
-import dto.OutputLottosDto;
 import java.util.List;
-import validator.LottoValidator;
+import constants.ErrorMessage;
 
 public class Lotto {
     private final List<Integer> numbers;
 
     public Lotto(List<Integer> numbers) {
         this.numbers = numbers.stream().sorted().toList();
-        LottoValidator.validate(this.numbers);
+        validate(this.numbers);
```

Comment:
생성자에서 검증을 할 땐 보통 할당하기 전에 검증함수를 호출한답니다! 😄 
```suggestion
        validate(this.numbers);
        this.numbers = numbers.stream().sorted().toList();
```

Reply:
앗 저도 바꾸면서 뭔가 이러면 안될 것 같다는 느낌이 들긴 했었는데..! 저렇게 작성한 이유는 validate에 정렬을 검증하는 로직이 포함되어 있기에, 정렬을 먼저 하고 validate를 해야만 했어요.

그러면 검증 함수를 먼저 호출하고, 검증 함수에서 정렬 체크하는 부분을 제외하는 것도 좋은 방법일까요?

Reply:
정렬이 중요한 도메인 검증조건이라면 정렬 후 검증을 해야합니다.
그러나 로또 번호가 정렬되어 보여지는건 View의 요구사항 같아보이고 정렬된 채로 유지되어야 한다라는 도메인 요구사항은 아직 없어보여서 말씀드렸어요!

Reply:
그렇네요. 사실 정렬이라는 건 도메인이 신경 쓸 사항이 아니었네요.
그러면 정렬하는 함수도 domain이 아니라 view에 있는 게 조금 더 자연스러울까요?

Reply:
네! 저는 정렬하는 것은 view의 요구사항이라 view에 있는게 자연스럽다고 생각합니다 😄